de.dlr.sc.overtarget.language.ui/src/de/dlr/sc/overtarget/language/ui/quickfix/OvertargetQuickfixProvider.xtend

```diff
@@ -48,16 +51,31 @@ class OvertargetQuickfixProvider extends DefaultQuickfixProvider {
 	
 	@Fix(Diagnostic.LINKING_DIAGNOSTIC)
 	def fixCannotResolveReference(Issue issue, IssueResolutionAcceptor acceptor) {
-		acceptor.accept(issue, 'Generate Target for References', '', '', 
+		acceptor.accept(issue, 'Generate targetForReferences to resolve references', 'Generates a targetForReferences and sets it as active target. \n After that the actual target, which could not be resolved, is set as active target.', '', 
```

Comment:
```suggestion
		acceptor.accept(issue, 'Use temporary target to to resolve TModel references', 'Generates a temporary target for resolving TModel references and sets it as active target. \n After resolving the references of this target, it is set as active target again.', '', 
```

Reply:
I think no one will understand what a "targetForReferences" is and what it does.